client/components/list-end/style.scss

```diff
@@ -1,11 +1,11 @@
 /**
-* Infinite Scroll End:
+* List End:
 * ------
 * Marker that gets appended to the end of a list (e.g. posts) once
 * infinite scroll has reached the last page.
```

Comment:
Could probably update this to drop reference to infinite scroll. The whole file could do for some reformatting to correctly use tabs as well.